internal/terraform/resource.go

```diff
@@ -70,5 +72,5 @@ type resourcesSortedByType []*Resource
 func (a resourcesSortedByType) Len() int      { return len(a) }
 func (a resourcesSortedByType) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
 func (a resourcesSortedByType) Less(i, j int) bool {
-	return a[i].FullType() < a[j].FullType() || (a[i].FullType() == a[j].FullType() && a[i].FullType()+a[i].Mode <= a[j].FullType()+a[j].Mode)
+	return a[i].FullType() < a[j].FullType() || (a[i].FullType() == a[j].FullType() && a[i].Mode <= a[j].Mode && a[i].Name <= a[j].Name)
```

Comment:
Please add items to `sampleResources` and corresponding tests to `resource_test.go`

Reply:
@khos2ow para alcançar esse objetivo dos testes, eu precisei usar os valores diretos do objeto ao invés de usar a função `FullType`, está ok?

Reply:
Yes, that looks good!